### ISSUE TITLE
Format option is not correctly applied for SELECT query with WITH clause 

### ISSUE THREAD

**Issue by jirisamek** [apla/node-clickhouse#56]
### Example:
```ClickHouse({format: "JSON"})```
Query:
```
WITH sum(bytes) as s
SELECT
    formatReadableSize(s),
    table
FROM system.parts
GROUP BY table
ORDER BY s
```
### Current  behaviour:
Execution of the query produces result formatted as plaintext.
### Expected behaviour:
Execution of the query produces result formatted JSON.

### Workaround: 
Explicitly add FROMAT JSON to the query.
```
WITH sum(bytes) as s
SELECT
    formatReadableSize(s),
    table
FROM system.parts
GROUP BY table
ORDER BY s
FORMAT JSON
```

**Comment by evle** [comment]
@nezed same issue here

**Comment by nezed** [comment]
Fixed with #66
